libs/datasets/manual_filter.py

```diff
@@ -240,14 +240,46 @@ def _transform_one_override(
     override: Mapping, cbsa_to_counties_map: Mapping[Region, List[Region]]
 ) -> Filter:
     region_str = override["region"]
+    include_str = override["include"]
+
+    if not isinstance(region_str, str):
+        raise ValueError(f"Invalid 'region': {region_str}")
+    if not isinstance(include_str, str):
+        raise ValueError(f"Invalid 'include': {include_str}")
+
+    # Split on commas.
+    region_strs = [single_region_str.strip() for single_region_str in region_str.split(",")]
```

Comment:
How about doing this parsing of the region_str in `_transform_region_str`?

Reply:
Hmm... `list(chain.from_iterable(map(_transform_region_str, region_str.split(','))))` is pretty deeply nested but not horrible.

Reply:
Ah, nice.  I opted to leave it as-is, but thanks for the pointer.